src/Sylius/Bundle/OrderBundle/Resources/translations/flashes.en.yml

```diff
@@ -4,8 +4,5 @@
 sylius:
     cart:
         cart_save_completed: The cart has been successfully updated.
-        cart_clear_completed: The cart has been successfully cleared.
         item_add_completed: Item has been added to cart.
         item_remove_completed: Item has been removed from cart.
```

Comment:
These flashes are probably not used in bundle, have we decided where to put them?